mkosi.build

```diff
@@ -27,7 +27,7 @@
 export LC_CTYPE=C.UTF-8
 test -f "$BUILDDIR"/build.ninja || meson "$BUILDDIR"
 ninja -C "$BUILDDIR" all
-ninja -C "$BUILDDIR" test
+[ "$WITH_TESTS" = 1 ] && ninja -C "$BUILDDIR" test
```

Comment:
That means that with older mkosi versions tests are now not run at all. I'd rather invert the condition to

    [ "$WITH_TESTS" = 0 ] || ninja -C "$BUILDDIR" test

This is backwards compatible, and also avoids the entire command to fail if the condition is false (`false && true` is false, so `&&` should generally be avoided in `set -e` scripts; `||` is much better).